src/main/java/com/epam/aidial/core/util/ProxyUtil.java

```diff
@@ -161,50 +161,80 @@ private static void collectAttachedFilesCustomInput(JsonNode input, Consumer<Str
         }
     }
 
-    private static void collectAttachedFilesChatCompletion(ObjectNode tree, Consumer<String> consumer) {
-        ArrayNode messages = (ArrayNode) tree.get("messages");
-        if (messages == null) {
+    private static void collectAttachedFilesFromContent(JsonNode content, Consumer<String> consumer) {
+        if (content == null || !(content instanceof ArrayNode)) {
```

Comment:
remove redundant check `content == null`

Reply:
it could be replaced with

```
if (!(content instanceof ArrayNode contentParts)) {
            return;
        }
```

Reply:
fixed